src/plugins/vis_type_timeseries/public/actions/last_value_mode_action.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { EmbeddableInput, EmbeddableOutput, IEmbeddable } from '../../../embeddable/public';
+import { createAction } from '../../../ui_actions/public';
+import { TIME_RANGE_DATA_MODES } from '../../common/timerange_data_modes';
+import { PANEL_TYPES } from '../../common/panel_types';
+import { Vis } from '../../../visualizations/public';
+
+const LAST_VALUE_MODE = 'lastValueMode';
+
+interface TimeseriesEmbeddableOutput extends EmbeddableOutput {
+  visTypeName?: string;
+}
+
+interface TimeseriesEmbeddable extends IEmbeddable<EmbeddableInput, TimeseriesEmbeddableOutput> {
+  vis: Vis;
+}
+
+interface LastValueModeActionContext {
+  embeddable: TimeseriesEmbeddable;
+}
+
+export const lastValueModeAction = createAction<LastValueModeActionContext>({
+  id: LAST_VALUE_MODE,
+  getDisplayName: () =>
+    i18n.translate('visTypeTimeseries.lastValueModeAction.lastValueModeName', {
+      defaultMessage: 'Last value',
+    }),
+  isCompatible: async ({ embeddable }: LastValueModeActionContext) => {
+    if (
+      embeddable.getOutput().visTypeName === 'metrics' &&
+      embeddable.vis.params.type !== PANEL_TYPES.TIMESERIES
+    ) {
+      const timeRangeMode = embeddable.vis.params.time_range_mode;
+      return !timeRangeMode || timeRangeMode === TIME_RANGE_DATA_MODES.LAST_VALUE;
+    }
+    return false;
+  },
+  execute: async () => {},
+});
```

Comment:
@streamich could you please take a look?
during the [discussion](https://github.com/elastic/kibana/pull/91977#discussion_r585996022), it's been decided to add an indicator on top of the embeddable to indicate if visualization is using last value mode 
so what do you think about this solution? would appreciate any comments or other ideas how to implement that

Reply:
i am missing some context, what is this supposed to do ? action without execute ? or is this work in progress ?

Reply:
so after being explained what we need i think we should try to put the message inside TSVB (instead of panel header).
currently panel headers are showing things that are common for all the embeddables and i think it should stay that way.

if we really wanted to show some indication like this inside the panel header we need to expand the embeddable API to support this, which will take some time as we'll want to take it thru planning, rfc and arch review.